README.md

```diff
@@ -149,6 +155,32 @@ subtract_omit = */.tox/*
 this will result in `*/.tox/*` not being `omit`ted (`*/.tox/*` is among the
 defaults provided by `covdefaults`).
 
+#### coverage for installed libraries
+
+By default ``covdefaults`` assumes that you want to track coverage
+in your local source tree, code that is version controlled. For libraries
+it's better idea to detect coverage on your installed version, and map that
+back to the source files. You can enable tracking installed libraries via:
+
+```ini
+[coverage:covdefaults]
+installed_libraries = tox:src virtualenv:.
+```
+
+In this example we say we'll track the installed package/module tox that maps
+back to the source folder present at ``src``, and ``virtualenv`` that maps back
+to the source folder present at ``.``. Note the installed package/module must be
+present in either the ``platlib`` or ``purelib`` path of the virtual environment
+as specified by ``distutils`` ``install`` object.
```

Comment:
let's not involve distutils, that severely limits the portability (debian) -- we can get this data directly from `sysconfig`

Reply:
I'm aware of multiple instances where sysconfig differs from distlib - sysconfig generally contains configuration for the system python, distlib contains information about the current python (aka can differ in virtual environments); pip and virtualenv is using distlib.... so if we go down the sysyconfig path this will be broken in all those cases.

Reply:
seems to work fine for me:

```pycon
>>> sysconfig.get_paths()['purelib']
'/tmp/vvv/lib/python3.6/site-packages'
>>> sysconfig.get_paths()['platlib']
'/tmp/vvv/lib/python3.6/site-packages'
```

could you enumerate those cases?

Reply:
I don't have it available from the top of my head, but I remember there were some platforms (though they might be old/unsupported now), and pip also uses distlib to determine where to install (always), so felt safer to use that either way.

Reply:
until there's a demonstrated problem I'd prefer the simplest thing that works